templates/zerver/integrations.html

```diff
@@ -2940,6 +2940,93 @@
       their own Zulip account email in the webhook URL.</p>
     </div>
 
+    <div id="wordpress" class="integration-instructions">
+
+      <p>See WordPress blog notifications in Zulip!</p>
```

Comment:
The existing style in the original file was far from consistent, were all the new doc.html files reformatted to match? I can certainly do that.